test/e2e/specs/site-editor/template-parts-mode.spec.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Template Parts for Classic themes', () => {
```

Comment:
Great idea to have an e2e test for this, we don't check this flow often manually.